### ISSUE TITLE
Capability support for get_property and get_attribute functions

### ISSUE THREAD

**Issue by alexmaslenn** [xlab-si/xopera-opera#58]
## Description
As defined in the TOSCA standart **get_property** and **get_attribute** functions have an optional parameter <optional_req_or_cap_name> which is
> The optional name of the requirement or capability name within the modelable entity

Currently xOpera supports returning property and attribute values from requirements but not from capabilities.
## Prerequisites
- `opera` installed
- `service.yaml` TOSCA Service Template containing node type that has a reference to capability name in **get_property** function, e.g. in operation definition
```
node_type_name:
  derived_from: tosca.nodes.Root
  capabilities:  
    capability_name:  
      type: capability_type
  interfaces:  
    Standard:  
      type: tosca.interfaces.node.lifecycle.Standard 
      operations:
        create:  
          inputs:  
            input_name: { default: { get_property: [ SELF, capability_name, property_name] } } 
```
where `property_name` value is defined in capability assignment within node template definition
```
node_template_name:
  type:  node_type_name
  capabilities:  
    capability_name:  
      properties:
        property_name: some_value
```

## Steps
- User runs deploy `opera deploy service.yaml`
## Current behavior
 - xOpera returns an error `Cannot find property 'capability_name'.`
## Expected behavior
- Service template is deployed without error.

**Comment by anzoman** [comment]
Good call @alexmaslenn! Great that you noticed that the support for getting properties from capabilities is missing and this issue will remind us that implementing this should not be waved aside.

**Comment by anzoman** [comment]
@alexmaslenn I have tried to implement this new feature and have created a PR: #73. So you can take a look and express your thoughts on that.

**Comment by anzoman** [comment]
This was successfully implemented with #73 and now we can close this issue. :whale:
